src/components/organisms/UiMultipleChoices/_internal/UiMultipleChoicesItem.vue

```diff
@@ -308,107 +351,56 @@ const optionsToRender = computed(() => props.options.map((option) => ({
+    --list-item-content-padding: #{functions.var($element + "-content", padding, 0)};
+    --list-item-content-hover-background: #{functions.var($element + "-content-hover", background, transparent)};
 
-      display: block;
-    }
+    display: block;
 
     @include mixins.from-tablet {
+      --list-item-content-hover-background: #{functions.var($element + "-tablet-content-hover", background)};
+
       display: flex;
-      flex-direction: row;
-      justify-content: space-between;
+      gap: functions.var($element + "-tablet-content", gap, var(--space-24));
     }
   }
 
   &__header {
     display: flex;
     justify-content: space-between;
     padding: functions.var($element + "-header", padding, var(--space-12) var(--space-20));
+    margin: functions.var($element + "-header", margin, var(--space-20) 0 0 0);
+    gap: functions.var($element + "-header", gap, var(--space-12));
 
     @include mixins.from-tablet {
-      flex: 1;
       flex-direction: column;
       align-items: flex-start;
-      justify-content: flex-start;
       padding: functions.var($element + "-tablet-header", padding, 0);
-    }
-  }
-
-  &__options {
-    @include mixins.from-tablet {
-      display: flex;
-      flex-direction: row;
-    }
-  }
-
-  &__option {
-    --list-item-content-padding: #{functions.var($element + "-option", padding, var(--space-12) var(--space-20))};
-    --list-item-tablet-content-padding: #{functions.var($element + "-tablet-option", padding, 0)};
-    --list-item-border-width: #{functions.var($element + "-option", border-width, 1px 0 0)};
-    --list-item-content-hover-background: #{functions.var($element + "-option-hover", background, var(--color-background-white-hover))};
-
-    &:last-of-type {
-      --list-item-border-width: #{functions.var($element + "-option", border-width, 1px 0)};
-    }
-
-    @include mixins.from-tablet {
-      --list-item-border-width: #{functions.var($element + "-tablet-option", border-width, 0)};
-      --list-item-content-hover-background: #{functions.var($element + "-tablet-option-hover", background, transparent)};
-      --radio-label-margin: 0 0 0 var(--space-8);
-
-      display: flex;
-      flex-direction: row;
-      margin: functions.var($element + "-tablet-option", margin, 0 0 0 var(--space-24));
-
-      &:first-of-type {
-        margin: functions.var($element + "-tablet-option", margin, 0);
-      }
-
-      &:last-of-type {
-        --list-item-border-width: #{functions.var($element + "-tablet-option", border-width, 0)};
-      }
-
-      [dir="rtl"] & {
-        margin: functions.var($element + "-rtl-tablet-option", margin, 0 var(--space-24) 0 0);
-
-        &:first-of-type {
-          margin: functions.var($element + "-rtl-tablet-option", margin, 0);
-        }
-      }
+      margin: functions.var($element + "-tablet-header", margin, 0);
+      gap: functions.var($element + "-tablet-header", gap, var(--space-8));
```

Comment:
@ArkadiuszKrzymienski thanks for showing me how `gap` is useful. I love it now <3